gossipd/gossip.c

```diff
@@ -1769,6 +1769,28 @@ static struct io_plan *handle_txout_reply(struct io_conn *conn,
 	return daemon_conn_read_next(conn, &daemon->master);
 }
 
+static struct io_plan *routing_failure_req(struct io_conn *conn,
+					   struct daemon *daemon,
+					   const u8 *msg)
+{
+	struct pubkey erring_node;
+	struct short_channel_id erring_channel;
+	u16 failcode;
```

Comment:
Wondering whether this can be `enum onion_type` right away. Would save us the cast down below.

Reply:
`onion_type` itself seems generated, and does not seem to play nice when used in a csv file source.  Or maybe some other way?

Reply:
I think it works, just need with #include <wire/gen_onion_wire.h> in the CSV file.